frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -776,5 +786,43 @@ export const eventUsageLogic = kea<
         reportRecordingPlayerSkipInactivityToggled: ({ skipInactivity }) => {
             posthog.capture('recording player skip inactivity toggled', { skip_inactivity: skipInactivity })
         },
+        reportNewExperimentButtonClicked: () => {
+            posthog.capture('new experiment button clicked')
```

Comment:
Perhaps this one we want to rely on autocapture? Or Pageview on `/new` ? You could've come up from a direct link.

Reply:
Makes sense. I want to use this to track abandonment, and pageview on `/new` works nicely.